bayesian/utility/user_utils.py

```diff
@@ -22,42 +26,59 @@ def get_user(uuid_val):
         user = result.one()
         return user
     except SQLAlchemyError as e:
-        logger.exception(f"Error fetching user with id {uuid_val}")
+        logger.exception("Error fetching user with id %s", uuid_val)
         raise UserException("Error fetching user") from e
 
 
 @retry(reraise=True, stop=tenacity.stop_after_attempt(3), wait=tenacity.wait_fixed(1))
-def create_user(uuid_val, user_source):
-    """Create User."""
+def create_or_update_user(user_id, snyk_api_token, user_source):
```

Comment:
Can we move this to `db_gateway.py`

Reply:
I am not sure if we want all DB interactions to go through that. That file would then being to become very big.

Reply:
I leave it up to you. :)